intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -171,6 +171,7 @@ \subsection{Pre-defined user directories}
                          shared across all clusters of the VSC in the future. See \S\ref{subsec:scratch-directory}. \\ \hline
 \ifgent
 \$VSC\_SCRATCH\_CLUSTER & The scratch filesystem closest to this cluster.\\ \hline
+\$VSC\_SCRATCH\_PHANPY  & The scratch filesystem on phanpy. Since phanpy has SSDs, this is faster than other scratch filesystems.\\ \hline
```

Comment:
`A separate (smaller) shared scratch filesystem, powered by SSDs.`

(are SSDs mentioned in the glossary?)

Reply:
Maybe also add: `intended for very I/O-intensive workloads.`